### PR TITLE
Handle weird node parsing

### PR DESCRIPTION
**Pull request by matks**
The following payload is badly parsed and causes catalog extraction to fail:
```
        $useStaticLoader = PHP_VERSION_ID >= 50600 && !defined('HHVM_VERSION') && (!function_exists('zend_loader_file_encoded') || !zend_loader_file_encoded());
```

I added it to the test cases and added the fix.

### CODE REVIEW

**Comment by matks** [comment]
Thanks for review @Quetzacoalt91

File: Tests/resources/fixtures/TestController.php [hunk 1]
```diff
@@ -11,6 +11,8 @@ public function init()
 
     public function productAction()
     {
+        $useStaticLoader = PHP_VERSION_ID >= 50600 && !defined('HHVM_VERSION') && (!function_exists('zend_loader_file_encoded') || !zend_loader_file_encoded());
+
```
Comment (eternoendless) [inline Tests/resources/fixtures/TestController.php]: This code should never be executed, why did you add it?

File: Translation/Extractor/Visitor/Translation/ExplicitTranslationCall.php [hunk 2]
```diff
@@ -21,7 +21,7 @@ public function leaveNode(Node $node)
      */
     public function extractFrom(Node $node)
     {
-        if (!$this->appliesFor($node)) {
+        if (empty($node->args) || !$this->appliesFor($node)) {
```
Comment (eternoendless) [inline Translation/Extractor/Visitor/Translation/ExplicitTranslationCall.php]: This change requires a test case
